package.json

```diff
@@ -23,6 +23,7 @@
     "deploy:package": "scripts/publish.sh",
     "commitmsg": "commitlint -e $GIT_PARAMS",
     "dev": "styleguidist server --config config/styleguide.config.js",
+    "dev:e2e": "STYLEGUIDIST_ENV=e2e styleguidist server --config config/styleguide.config.js",
```

Comment:
This change makes it more necessary for the `yarn e2e` test start and stop the styleguide too, as it seems likely that we will forget this step.

The `yarn e2e` script should:

1. start the docs server with the "e2e" environment
2. wait for it to fully start
2. run tests
3. close down the docs server, whether the tests passed, failed, or the user exited the process.

All of that is out of scope here for sure. Perhaps you can create a story for that?